Automattic/jetpack
# Warm Welcome: Update illustrations across all paid plan welcome screens

Resolves #9143 

This PR updates the illustrations in the welcome screens on all Jetpack plan levels need to be more Jetpack-ified (new brand green #00be28), and illustrations on gray backgrounds are updated to fill in incomplete white/color fills.

Before/After of Personal, Premium, and Professional plan welcome screens:

<img width="736" alt="screen shot 2018-03-28 at 9 27 48 pm" src="https://user-images.githubusercontent.com/204742/38068548-0245138c-32cf-11e8-8c40-d13b2674e502.png">
<img width="737" alt="screen shot 2018-03-28 at 9 28 00 pm" src="https://user-images.githubusercontent.com/204742/38068549-0256d32e-32cf-11e8-88a2-49e9a6196aab.png">
<img width="739" alt="screen shot 2018-03-28 at 9 28 11 pm" src="https://user-images.githubusercontent.com/204742/38068550-02693d3e-32cf-11e8-859a-8684be044c5c.png">


**Testing Instructions:**
* Check out this branch
* Run yarn build
* Visit the Jetpack dashboard in wp-admin on a new site that has the Personal plan and verify the updated welcome dialog box has the updated "After" illustrations as in the above screenshots.
* Visit the Jetpack dashboard in wp-admin on a new site that has the Premium plan and verify the updated welcome dialog box has the updated "After" illustrations as in the above screenshots.
* Visit the Jetpack dashboard in wp-admin on a new site that has the Professional plan and verify the updated welcome dialog box has the updated "After" illustrations as in the above screenshots.

**Changelog entry**
Updated new paid plan welcome screen illustrations to include Jetpack's new brand colors.
